assets/javascripts/kitten/components/cards/crowdfunding-card/index.js

```diff
@@ -10,110 +10,118 @@ import classNames from 'classnames'
-          />
-          {additionalInfo && (
-            <span className="k-CrowdfundingCard__additionalInfo">
-              <Text size="micro" lineHeight="normal" title={additionalInfo}>
-                {additionalInfo}
-              </Text>
-            </span>
-          )}
-        </div>
-        <Informations
-          info1={info1}
-          info2={info2}
-          info3={info3}
+  return (
+    <StyledCrowdfundingCard
+      {...others}
+      as={href ? 'a' : 'div'}
+      imageContainerRatio={imageContainerRatio}
+      onClick={removeCurrentFocus}
+      className={classNames(
+        'k-CrowdfundingCard',
+        'k-Card k-Card--light k-Card--withoutBoxShadowOnHover',
+        className,
+        {
+          'k-CrowdfundingCard--titlesMinHeight': titlesMinHeight,
+          'k-CrowdfundingCard--isLoading': loading,
+          'k-CrowdfundingCard--isStretched': stretch,
+        },
+      )}
+      href={href}
+    >
+      <Image
+        ownerTitle={ownerTitle}
+        ownerDescription={ownerDescription}
+        imageContainerBackground={imageContainerBackground}
+        imageProps={imageProps}
+        avatarProps={avatarProps}
+        loading={loading}
+      />
+      <div className="k-CrowdfundingCard__titleAndDesc">
+        <TitleComponent
+          title={cardTitle}
           loading={loading}
+          titleTruncate={titleTruncate}
         />
-        <ProgressBar
-          progress={progress}
-          progressColor={progressColor}
+        <Subtitle
+          subTitle={cardSubTitle}
+          subTitleTruncate={subTitleTruncate}
           loading={loading}
         />
-        <State state={state} widgetState={widgetState} loading={loading} />
+        {!loading && additionalInfo && (
+          <span className="k-CrowdfundingCard__additionalInfo">
+            <Text
+              size="micro"
+              lineHeight="normal"
+              weight="light"
+              title={additionalInfo}
+            >
+              {additionalInfo}
+            </Text>
+          </span>
+        )}
+      </div>
+      <Informations
+        info1={info1}
+        info2={info2}
+        info3={info3}
+        loading={loading}
+      />
+      <ProgressBar
+        progress={progress}
+        progressColor={progressColor}
+        loading={loading}
+      />
+      <State state={state} widgetState={widgetState} loading={loading} />
 
-        {loading && <span className="k-CrowdfundingCard__loading" />}
-      </StyledCrowdfundingCard>
-    )
-  }
+      {loading && <span className="k-CrowdfundingCard__loading" />}
+    </StyledCrowdfundingCard>
+  )
+}
+
+CrowdfundingCard.propTypes = {
+  href: PropTypes.string,
+  titlesMinHeight: PropTypes.bool,
+  stretch: PropTypes.bool,
+  imageContainerRatio: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]),
+}
+
+CrowdfundingCard.defaultProps = {
+  href: null,
+  titlesMinHeight: true,
+  stretch: false,
+  imageContainerRatio: 10 / 16,
```

Comment:
```suggestion
  imageContainerRatio: 10 / 16,
```

Ça ne me parait pas tellement logique de passer un ratio inverse de celui qu'on cherche. Quand on parle d'un ratio c'est toujours largeur / hauteur. C'est beaucoup plus lisible, on n'a pas à se poser de question quand on intègre le composant dans une app. Du 16/9 ? j'écris 16/9. C'est à l'app de faire le calcul après, tu penses pas ?

Reply:
J'ai mis un calcul par défaut pour avoir la valeur exacte. Comment tu peux faire le discernement entre un calcul et une valeur écrite directement : `0.625` ? Il n'y a aucun moyen :). Je ne suis pas fan de diviser en 2 props le ratio. Je pense qu'on devrait laisser tel quel. À l'usage, on verra bien qu'une des 2 valeurs doit être inversée si le ratio n'est pas bon.